prow/config/istio-private_jobs/release-builder-1.16.yaml

```diff
@@ -34,6 +34,7 @@ transforms:
     GCS_BUCKET: istio-private-prerelease/prerelease
     HELM_BUCKET: istio-private-build/dev/charts
     PRERELEASE_DOCKER_HUB: gcr.io/istio-prow-build
+    GCP_SECRETS: null # Don't use their secrets. We only need to push to GCS and GCR which do not need secrets
```

Comment:
I would expect release-builder to require github creds to fetch repos and dependencies

Reply:
Good call, forgot about that. Looks like its setup to use `ssh-key-secret`. This is handled by prow's containers so we cannot use the GCP_SECRETS stuff. Not a big deal though, we can use a secret

Reply:
Still thinking about how to handle this best. I will handle in a followup if necessary, I am sure we will need a bit of iteration before this is working end to end